frontend/public/index.html

```diff
@@ -53,6 +53,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script type="text/javascript">
       window.SERVER_FLAGS = [[.]];
+      const htmlTagElement = document.documentElement;
+      const theme = localStorage.getItem('bridge/theme') || 'systemDefault';
+      let systemTheme;
+      if (theme === systemDefault && window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
```

Comment:
This should compare theme against the string `'systemDefault'`, right?

```suggestion
      if (theme === 'systemDefault' && window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
```